internal/pkg/describe/env.go

```diff
@@ -50,23 +60,58 @@ func NewEnvDescriber(projectName string, envName string) (*EnvDescriber, error)
 		return nil, fmt.Errorf("assuming role for environment %s: %w", env.ManagerRoleARN, err)
 	}
 	return &EnvDescriber{
-		env:          env,
-		store:        svc,
-		apps:         apps,
-		sessProvider: sess,
+		env:            env,
+		store:          svc,
+		proj:           proj,
+		apps:           apps,
+		sessProvider:   sess,
+		rgClient:       resourcegroups.New(sess),
+		cloudformation: cloudformation.New(sess),
 	}, nil
 }
 
 // Describe returns info about a project's environment.
 func (e *EnvDescriber) Describe() (*EnvDescription, error) {
-	var tags map[string]string
+	appsForEnv, err := e.FilterAppsForEnv()
+	if err != nil {
+		return nil, err
+	}
+
 	return &EnvDescription{
 		Environment:  e.env,
-		Applications: e.apps,
-		Tags:         tags,
+		Applications: appsForEnv,
+		Tags:         e.proj.Tags,
 	}, nil
 }
 
+func (e *EnvDescriber) FilterAppsForEnv() ([]*archer.Application, error) {
+	var appObjects []*archer.Application
+	tags := map[string]string{
+		"ecs-environment": e.env.Name,
+	}
+	stackNames, err := e.rgClient.GetResourcesByTags(cloudformationResourceType, tags)
+	if err != nil {
+		return nil, err
+	}
+	for _, singleStack := range stackNames {
+		stackDesc, err := e.cloudformation.Describe(singleStack)
+		if err != nil {
+			return nil, err
+		}
+		params := make(map[string]string)
+		for _, param := range stackDesc.Parameters {
+			params[*param.ParameterKey] = *param.ParameterValue
+		}
+		appName := params["AppName"]
+		for _, app := range e.apps {
```

Comment:
Why do we need to do this :P We can prob just include those deployed app name if we include `"ecs-project": e.env.Project` when searching for resources.

Reply:
I was trying to get archer.Application objects from the stack strings returned by `GetResourcesByTags`. How do I more efficiently extract the app name from the stack string and convert it into the app object (so we can access the fields Name and Type for output)? Thanks!

Reply:
Oh I got you. Then it makes sense to me.